src/components/PageTreeViewItemActions.tsx

```diff
@@ -47,28 +47,47 @@ export const PageTreeViewItemActions = ({ page, onActionOpen, onActionClose }: P
       navigateUrl({ path });
     }
   }, [newPage, navigateUrl, resolveIntentLink]);
+
+  const menuButtons = config.pageSchemaTypes
+    .filter(
+      type =>
+        type !== config.rootSchemaType &&
+        (config.allowedParents?.[type] === undefined || config.allowedParents?.[type]?.includes(page._type)),
+    )
+    .map(type => (
+      <MenuItem key={type} onClick={() => onAdd(type)} text={schema.get(type)?.title ?? type} value={type} />
+    ));
+
+  const isAddPageButtonDisabled = menuButtons.length === 0;
+  const tooltipContent = isAddPageButtonDisabled ? (
+    <Box padding={1}>
+      <Text muted size={1}>
+        The allowed parents configuration prevents this document type from being a parent to any other document.
```

Comment:
I think this text is just a little bit too technical for the average content editor. How about `This page cannot contain any child pages.` or something similar?